packages/db/src/methods/helpers/deleteOneFrom.js

```diff
@@ -0,0 +1,10 @@
+const R = require('ramda')
+
+const getOneFrom = (knex, tableName, selector) =>
+  knex(tableName)
+    .where(selector)
+    .limit(1)
+    .del()
+    .then(R.head) //
```

Comment:
Not useful response, should indicate if deleted or not, and maybe return deleted row